tofhir-engine/src/main/scala/io/tofhir/engine/model/FhirMapping.scala

```diff
@@ -12,6 +12,7 @@ import java.util.UUID
  * @param url         Canonical url for the mapping
  * @param name        Computer friendly name
  * @param title       Human friendly title
+ * @param isDraft     Indicate whether mapping is unready to execution
```

Comment:
Let's change its comment to: **Indicates whether the mapping is in a draft state and not yet ready for execution**